courses-app/src/app/shared/components/password-eye/password-eye.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { PasswordEyeComponent } from './password-eye.component';
+
+describe('PasswordEyeComponent', () => {
+  let component: PasswordEyeComponent;
+  let fixture: ComponentFixture<PasswordEyeComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ PasswordEyeComponent ]
```

Comment:
In testing, don't just include component in declaration, instead use whole module you created for the component. Read about  SCAM:

https://medium.com/marmicode/your-angular-module-is-a-scam-b4136ca3917b